cmd/agent/local.go

```diff
@@ -8,11 +8,12 @@ import (
 	"github.com/gravitational/satellite/monitoring"
 )
 
-func localChecks() {
+func localChecks() error {
```

Comment:
why this since it never returns an error - in preparation of future changes or to make it look symmetrical to `status`?

Reply:
yes, consistency

Reply:
i mean, why invent the error if one is not there?..